tests/installation/disk_activation_iscsi.pm

```diff
@@ -0,0 +1,72 @@
+# SUSE's openQA tests
+#
+# Copyright © 2016-2019 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+
+# Summary: initial installation: iSCSI client for the supportserver
+#          supportserver to provide one iSCSI target which will 
+#          provide a multipathed second disk to the client.
+# Maintainer: Klaus Wagner <kgw@suse.com>
+
+use base 'y2_installbase';
+use strict;
+use warnings;
+use testapi;
+
+sub run {
+    my $self = shift;
+    my $iscsi_iqn = "iqn.2016-02.de.openqa";
+    my $iscsi_server_ip = "10.0.2.1";
+
+    if (get_var('WITHISCSI') and get_var('USE_SUPPORT_SERVER')) {
```

Comment:
I think check for `SUPPORT_SERVER` is not mandatory.

Reply:
OK, let me investigate on what needs to be done to accomplish this kind of generalization.

Reply:
Generalized in amended PR commit e150dcc (below) as proposed in the previous reviewer's comment.